aidants_connect_web/tests/test_views/test_service.py

```diff
@@ -134,40 +136,32 @@ def test_environment_variables_are_accessible(self):
         self.assertEqual(secret_key, "Everything is awesome")
 
 
-@tag("service")
+@tag("service", "coucou")
 class StatistiquesTests(TestCase):
     def setUp(self):
-        autorisation_justice = AutorisationFactory()
-
-        Journal.objects.create(
-            action="create_attestation",
-            initiator=autorisation_justice.aidant.full_string_identifier,
-            usager=autorisation_justice.usager.full_string_identifier,
-            demarche=autorisation_justice.demarche,
-            duree=autorisation_justice.duree_in_days,
-            autorisation=autorisation_justice.id,
+        mairie_de_houlbec = OrganisationFactory()
+        aidant_thierry = AidantFactory()
+        usager_homer = UsagerFactory()
+        mandat_houlbec_homer = MandatFactory(
+            organisation=mairie_de_houlbec, usager=usager_homer
         )
-        Journal.objects.create(
-            action="create_autorisation",
-            initiator=autorisation_justice.aidant.full_string_identifier,
-            usager=autorisation_justice.usager.full_string_identifier,
-            demarche=autorisation_justice.demarche,
-            duree=autorisation_justice.duree_in_days,
-            autorisation=autorisation_justice.id,
+        autorisation_justice_houlbec_homer = AutorisationFactory(
+            mandat=mandat_houlbec_homer
         )
+
         Journal.objects.create(
-            initiator=autorisation_justice.aidant.full_string_identifier,
-            usager=autorisation_justice.usager.full_string_identifier,
+            initiator=aidant_thierry.full_string_identifier,
+            usager=usager_homer.full_string_identifier,
             action="use_autorisation",
             demarche="justice",
-            autorisation=autorisation_justice.id,
+            autorisation=autorisation_justice_houlbec_homer.id,
         )
         Journal.objects.create(
-            initiator=autorisation_justice.aidant.full_string_identifier,
-            usager=autorisation_justice.usager.full_string_identifier,
+            initiator=aidant_thierry.full_string_identifier,
+            usager=usager_homer.full_string_identifier,
             action="use_autorisation",
             demarche="justice",
-            autorisation=autorisation_justice.id,
+            autorisation=autorisation_justice_houlbec_homer.id,
         )
 
         # An Aidants Betagouv in in our mist !
```

Comment:
```suggestion
        # An Aidants Betagouv is among us!
```

Quoique... _in our mist_ a une certaine valeur poétique 😄